Readme.md

```diff
@@ -14,8 +14,9 @@ Do you want to quickly try out this project and see it in action? Check it out,
 
 - You must have [Node.js](https://nodejs.org/) installed and will need [PNPM](https://pnpm.io/) for the build to succeed.
   - Please use Node.js **v16.9** or above, or **v14.19** or above, which allow the usage of `pnpm` without prior installation.
```

Comment:
@Piedone Do you think it's necessary, or feasible, to mention NVM for Windows here? It seems to me that this is really a matter of preference and environment.